src/commands/init.ts

```diff
@@ -46,48 +43,55 @@ export default async (positionals: Array<string>): Promise<void> => {
     }
 
     const packages = await getLocalPackages();
-    const doModelsExist = (await getModels(packages)).length > 0;
+
+    // This case should never happen, since we login before running the init command
+    // if no tokens are provided.
+    if (!(token?.appToken || token?.sessionToken)) {
+      spinner.fail(
+        'Run `ronin login` to authenticate with RONIN or provide an app token',
+      );
+      process.exit(1);
+    }
+
+    const doModelsExist =
+      (
+        await getModels(
+          packages,
+          undefined,
+          token.appToken || token.sessionToken,
+          spaceHandle,
+        )
+      ).length > 0;
 
     if (doModelsExist) {
-      // Install the types package using the preferred package manager.
-      if (packageManager === 'bun') {
-        await exec(`bun add @ronin-types/${spaceHandle} --dev`);
-      } else {
-        await exec(`npm install @ronin-types/${spaceHandle} --save-dev`);
-      }
-    } else {
-      // Create ronin directories.
+      await types(token.appToken, token.sessionToken);
+    }
+
+    // Create ronin directories if they don't exist.
```

Comment:
```suggestion
    // Create migration directory if it doesn't exist.
```

The former seems very unclear.